### PR TITLE
Community - Add steem keychain

### PR DESCRIPTION
**Pull request by roadscape**
From @eonwarped #50:

> Essentially the analogue to https://github.com/steemit/condenser/pull/3382

### CODE REVIEW

File: src/app/redux/UserSaga.js [hunk 1]
```diff
@@ -152,6 +168,23 @@ function* usernamePasswordLogin({
...
     }
 
+    // return if already logged in using steem keychain
+    if (login_with_keychain) {
+        console.log('Logged in using steem keychain');
+        yield put(
+            userActions.setUser({
+                username,
+                login_with_keychain: true,
+                vesting_shares: account.get('vesting_shares'),
```
Comment (eonwarped) [inline src/app/redux/UserSaga.js]: This block needs to go below the fetch for account, missed it when i was implementing.

Comment (roadscape) [reply src/app/redux/UserSaga.js]: https://github.com/steemit/wallet/pull/64/commits/f11e4995cddc1f5e26b038edd96b3cd073553cbb

**Comment by roadscape** [comment]
Conflicts resolved w/ `master`. Needs more testing.

**Comment by gl2748** [comment]
@eonwarped 
This is great. So far I can sign in using the keychain just fine. However when I come to initiate a transfer, the operation seems to get stuck in the pending state, without any error being logged to the console. fig 2. Below shows the 'pending' state that it seems to get stuck on. I have tried this with other wallet operations like powering down etc and I see the same thing happen.

I would also add that for each individual transaction, it might make sense to prompt the user if they would like to use the Steem Keychain.

# 1.
Screen Shot 2019-10-25 at 10 40 26 AM

# 2.
Screen Shot 2019-10-25 at 10 40 33 AM

**Comment by gl2748** [comment]
@eonwarped nevermind, i just tried this on the updated PR here: https://github.com/steemit/wallet/pull/109 and it worked!

**Comment by roadscape** [comment]
Moving to bridge api. Thanks @eonwarped!

**Pull request closed without merging**